index.html

```diff
@@ -4,10 +4,15 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document</title> 
+    <script src="/idx"></script>
```

Comment:
You should add script at the bottom of your body or use defer attribute inside it so it will load at the end

Reply:
done👍